### PR TITLE
Documentation: Add a link list

### PR DESCRIPTION
**Pull request by kleinesfilmroellchen**
The link list tries to collect all links from the website, BenW's link
list <https://benwiederhake.github.io/serenity-fixmes/index.html#links>
and the unofficial wiki <https://wiki.serenityos.net/links.html> into
one sorted and expanded list, so that hopefully noone has difficulty
searching for serenity pages in the future :^)

### CODE REVIEW

File: Documentation/Links.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+-   [SonarCloud Static Analysis](https://sonarcloud.io/project/overview?id=SerenityOS_serenity)
+-   [libjs.dev](https://libjs.dev/)
+    -   [Try LibJS!](https://libjs.dev/repl/)
+-   [Compiler Explorer](https://serenity.godbolt.org/): Select "Lagom trunk" under "Libraries" and add the compiler option `-std=c++20`
+-   [LibFont.js](https://macdue.github.io/LibFont.js/)
+
+## Social
+
+-   [Discord Server](https://discord.gg/serenityos)
+-   [serenityos.social](https://serenityos.social/), the official Mastodon instance
```
Comment (timschumi) [inline Documentation/Links.md:30]: I know that the page says that it also lists unofficial sources (which, however, contradicts the "official" that is listed here), but considering Andreas' stance on it and that we are only accepting semi-recognized contributors there(?), I don't think it should be listed here.

File: Documentation/Links.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+-   [Map of Developers and Users](https://usermap.serenityos.org/)
+-   YouTube Channel of [Andreas Kling](https://www.youtube.com/@awesomekling), [Linus Groh](https://www.youtube.com/@linusgroh), [kleines Filmröllchen](https://www.youtube.com/@kleinesfilmroellchen), and [davidot](https://www.youtube.com/@davidot4475)
+-   Personal Blog of [Andreas Kling](https://awesomekling.github.io/), [Linus Groh](https://linus.dev/posts/), [Andrew Kaster](https://adkaster.github.io/), [Jesse Buhagiar](https://quaker762.github.io/), and [Sam Atkins](https://atkinssj.github.io/)
```
Comment (timschumi) [inline Documentation/Links.md:33]: I feel like these would look more nice in a sublist.

File: Documentation/Links.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    -   [Emoji Statistics](https://emoji.serenityos.net/chart.emoji.serenityos)
+-   [Flags](https://flags.serenityos.net/)
+-   [test262](https://libjs.dev/test262/) (JavaScript Spec Tests)
+-   [Wasm Spec Tests](https://libjs.dev/wasm/)
+-   [Test Performance](https://github.com/alimpfard/random-serenity-statistics/tree/main/view/benchmarks/x86_64)
+-   [serenityos.social Statistics](https://grafana.serenityos.social/public-dashboards/42f9a11380614367aa4851076ee6a66b?orgId=1)
+
+## Miscellaneous
+
+-   [Merch](https://teespring.com/stores/serenityos?utm_medium=product_shelf&utm_source=youtube&utm_content=YT-AGNkefCob-U_Mq4OnxSUagj77Nz8jYYCc-IuarARfod3YK0765pOaAwNayHAn5Mu6YTyuINDxjU33HWU_eb8Njg6x-XlSo_VxfBNcJFUWOheXkivDRfjrEn-p8J3GFJO7uh8T0qUoWqtdlJHviuwc1AdLRbDbltBe6VuFWlLFWrhihhUk0XmjQ%3D%3D)
```
Comment (timschumi) [inline Documentation/Links.md:53]: We should probably strip that metadata from the link. Also, don't we have `https://store.serenityos.org`?

File: Documentation/Links.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+-   [Lines of Code vs. Commits](https://github.com/alimpfard/random-serenity-statistics#random-serenityos-statistics)
+-   [CSS Conformance](https://css.tobyase.de/)
+-   [Fonts](https://fonts.serenityos.net/)
+-   [Emoji Table](https://emoji.serenityos.net/)
+    -   [Emoji Statistics](https://emoji.serenityos.net/chart.emoji.serenityos)
+-   [Flags](https://flags.serenityos.net/)
+-   [test262](https://libjs.dev/test262/) (JavaScript Spec Tests)
+-   [Wasm Spec Tests](https://libjs.dev/wasm/)
+-   [Test Performance](https://github.com/alimpfard/random-serenity-statistics/tree/main/view/benchmarks/x86_64)
+-   [serenityos.social Statistics](https://grafana.serenityos.social/public-dashboards/42f9a11380614367aa4851076ee6a66b?orgId=1)
```
Comment (timschumi) [inline Documentation/Links.md:49]: The things mentioned for `serenityos.social` above probably also apply for this.

File: Documentation/Links.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
+# SerenityOS-related pages across the web
+
+This is a roughly categorized list of pages relating to SerenityOS and its subprojects across the web. Many of these pages are "unofficial" in that some single person created them and/or maintains them independently; but we only list pages here that are widely recognized and of good quality. If you want your Serenity related page to be listed, just create a pull request adding it and we can discuss.
+
+## General
+
+-   **[serenityos.org](https://serenityos.org)**
+    -   [Manpages](https://man.serenityos.org/)
+    -   [1st](https://www.serenityos.org/happy/1st/), [2nd](https://www.serenityos.org/happy/2nd/), [3rd](https://www.serenityos.org/happy/3rd/), and [4th](https://www.serenityos.org/happy/4th/) Birthday Post
```
Comment (timschumi) [inline Documentation/Links.md:9]: How about this? Looks a bit more organized in my opinion.

```
- Birthday Posts
  - [1st](...)
  - [2nd](...)
  - [3rd](...)
  - [4th](...)
```

File: Documentation/Links.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+## Development
+
+-   [GitHub Organization](https://github.com/SerenityOS) and [GitHub Repositories](https://github.com/orgs/SerenityOS/repositories)
+-   [Changelog](https://changelog.serenityos.org/)
+-   [Issues Found by OSS-Fuzz Continous Fuzzing](https://bugs.chromium.org/p/oss-fuzz/issues/list?q=label:Proj-serenity)
+-   [Azure CI Overview](https://dev.azure.com/SerenityOS/SerenityOS/_build)
+-   [SonarCloud Static Analysis](https://sonarcloud.io/project/overview?id=SerenityOS_serenity)
+-   [libjs.dev](https://libjs.dev/)
+    -   [Try LibJS!](https://libjs.dev/repl/)
```
Comment (networkException) [inline Documentation/Links.md:23]: Maybe this should make clear that you are about to try LibJS online (like with copy.sh above)

File: Documentation/Links.md (hunk 4 above)
Comment (networkException) [reply Documentation/Links.md:49]: Also its probably better to use https://grafana.serenityos.social/public

File: Documentation/Links.md (hunk 1 above)
Comment (networkException) [reply Documentation/Links.md:30]: I think it would sense to be on the list but "official" isn't accurate

File: Documentation/Links.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+-   **[serenityos.org](https://serenityos.org)**
+    -   [Manpages](https://man.serenityos.org/)
+    -   Birthday Posts
+        -   [1st](https://www.serenityos.org/happy/1st/)
+        -   [2nd](https://www.serenityos.org/happy/2nd/)
+        -   [3rd](https://www.serenityos.org/happy/3rd/)
+        -   [4th](https://www.serenityos.org/happy/4th/)
+    -   [Emoji](https://emoji.serenityos.org/)
+-   [Ports](https://ports.serenityos.net/)
+-   [Try it out Online!](https://copy.sh/v86/?profile=serenity)
```
Comment (Hendiadyoin1) [inline Documentation/Links.md:16]: Maybe mention that this is an old i686 build, and not necessarily representative of the current state of some parts of the system

File: Documentation/Links.md [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+    -   [Flamegraph Distribution](https://benwiederhake.github.io/serenity-fixmes/flamegraph.html)
+-   [Lines of Code vs. Commits](https://github.com/alimpfard/random-serenity-statistics#random-serenityos-statistics)
+-   [CSS Conformance](https://css.tobyase.de/)
+-   [Fonts](https://fonts.serenityos.net/)
+-   [Emoji Table](https://emoji.serenityos.net/)
+    -   [Emoji Statistics](https://emoji.serenityos.net/chart.emoji.serenityos)
+-   [Flags](https://flags.serenityos.net/)
+-   [test262](https://libjs.dev/test262/) (JavaScript Spec Tests)
+-   [Wasm Spec Tests](https://libjs.dev/wasm/)
+-   [Test Performance](https://github.com/alimpfard/random-serenity-statistics/tree/main/view/benchmarks/x86_64)
```
Comment (linusg) [inline Documentation/Links.md:61]: I didn't even know this exists, wtf

Comment (kleinesfilmroellchen) [reply Documentation/Links.md:61]: that's like the whole point of creating this list lol

**Comment by linusg** [comment]
very cool, thx for making this!

**Pull request merged**